### PR TITLE
Support hardware loopback for ixiatg

### PR DESCRIPTION
**Pull request by alexmasi**

### CODE REVIEW

File: topo/node/keysight/keysight.go [hunk 1]
```diff
@@ -322,6 +322,12 @@ func (n *Node) FixInterfaces() {
 	}
 }
 
+func (n *Node) HardwareLoopbackSupported() bool {
```
Comment (bormanp) [inline topo/node/keysight/keysight.go:325]: After discussion with Alex we feel this is might not be the best name for this feature.  I am trying to think of better terminology but the only thing I have come up with us "BackToBack" vs "HardwareLoopback", but I am not at all set on that name.  Any other suggested names?  We can then look at the suggested names and select one.  LoopbackCable?  LoopbackLink?

**Review by bormanp — COMMENTED** [review]
The code looks fine.

File: topo/node/keysight/keysight.go (hunk 1 above)
Comment (alexmasi) [reply topo/node/keysight/keysight.go:325]: - DirectLoopback
- SelfLoop
- SelfLoopback

I don't know what this situation is called on real hardware routers so I'll defer to you

Comment (alexmasi) [reply topo/node/keysight/keysight.go]: settled on BackToBackLoop offline

**Review by bormanp — APPROVED** [review]

**Pull request merged**